### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by wintan1418**
this is a pull request for my just concluded bootstrap project

### CODE REVIEW

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required ⚠️ 

### 👍  @wintan1418 You've put in a lot of work in this project but you still need to make corrections before it can be approved.

## Required 🛠️ 

- [ ] The background color used in the header is not the same as yours
- [ ] The space on both left and right of the header is equal, unlike yours
- [ ] Increase the font of the subscribe button and bolden both the `subscribe` button and the `login` button

Original:
or_bhead

Yours:
bheader

- [ ] The button is inside the same box that houses the image
Original:
right1

Yours:
w1

- [ ] The navigation links are not well-aligned

- [ ] The polygon to the far right of the navbar should also be added

Original:
r_nav

Yours:
w_nav

- [ ] The tags `EMMYS`, `U.S.` and `EMMYS` are to be positioned relative to the image, not at the top

 Original:
ctr

Yours:
ctw

- [ ] On a smaller than screen, the` person icon`,` subscribe button` and `hamburger menu `should all be well spaced out

Original:
nav_ori

Yours:
nav_or2

- [ ] Fix the stylelint errors

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required ⚠️ 

Well done 💪  on implementing some of the required changes but there's still a little more to do before this project can be approved.

## [Required]

- [ ] The navigation links need to be stretched out the same way it is on the original page
- [ ] The search button and icon under the polygon should be beside the polygon
- [ ] Please, do away with the unnecessary space right below the navigation links and above the trending section

**Original:**
a1

**Yours**:
a0

- [ ] The button/tag has a very visible text. (You can get `ColorZilla`, a chrome extension used to pick colors from webpages.)
**Original**:
right1
**Yours**:
wrong

- [ ] The horizontal line covers both the subscribe and newsletter. You should also add a margin or padding before that section
**Original**:
newsl_1
**Yours**:
newsl_0

### Make the following changes and request another review

Happy Coding 💻 

Feel free to reach out to me on:
slack @ElijahTobs
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

File: README.md [hunk 1]
```diff
@@ -1 +1,36 @@
...
+👤 **Oluwadare Juwon**
+
+- GitHub: [@wintan1418](https://github.com/wintan1418)
+- Twitter: [@juwonoluwadare](https://twitter.com/oluwadarejuwon)
+- LinkedIn: [Oluwadare Juwon](https://www.linkedin.com/in/oluwadare-juwon-048a391a8/)
+
+
+
+## �� Contributing
+You can freely contribute to this project.
```
Comment (descholar-ceo) [inline README.md]: How to contribute to this project? This sentence `You can freely contribute to this project.` is not clear enough for a non-tech user to know how they contribute to your project, try to make it a bit clear

File: index.html [hunk 2]
```diff
@@ -0,0 +1,3077 @@
...
+  </script>
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-ho+j7jyWK8fNQe+A12Hb8AhRq26LrZ/JpcUGGOn+Y7RsweNrtN/tE3MoK7ZeZDyx" crossorigin="anonymous">
+  </script>
+
+  <!-- Option 2: jQuery, Popper.js, and Bootstrap JS
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js" integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/js/bootstrap.min.js" integrity="sha384-w1Q4orYjBQndcko6MimVbzY0tgp4pWB4lZ7lr30WKz0vr/aWKhXdBNmNb5D92v7s" crossorigin="anonymous"></script>
+    -->
```
Comment (descholar-ceo) [inline index.html]: If you don't need these codes, you should remove them completely, leaving commented codes from your codebase is a bad practice, try to fix that.

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Change Required :vertical_traffic_light: :warning: 
## Hello @wintan1418  :handshake: :handshake: :handshake: 
:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
## Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes]
- [ ] Newsweek logo size should change on different screen sizes, currently it's remaining on the same size and consequently, that's spoiling your design

- [ ] Your navbar needs more work, when the screen is between (995 px - 1325 px), your navbar on navigation section, it becomes a mess, you should fix that to make sure that the navigation links and search icon stay on the left side of the polygon, currently it's overlapping

| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-10-25 20-31-43| Screenshot from 2020-10-25 20-32-35|

:point_right: Could you see the difference :question: :question: :question: (Those screenshots are from a 998px screen)

- [ ] On medium screens and small screens, the navbar is overlapping the main page, fix that

| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-10-25 20-36-38 Could you see that the `top stories` title is hidden?| Screenshot from 2020-10-25 20-37-31|

- [ ] On small screens, the newsweek logo and the signin&subscribe sections are overlapping each other, please fix that as well

| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-10-25 20-40-45| Screenshot from 2020-10-25 20-41-44|

:point_right: Could you see the difference :question: :question: :question: 

- [ ] Try to not use id selectors in your css
- [ ] Try to use the same colors on the inline buttons in the editor's pick (text on the last box on the first row is not visible) :point_right:  this was requested in the previous review

- [ ] your footer looks good, but not the same as the original, try to use the same colors from the original footer, and fix the alignment issues 
- [ ] Don't put your license in .vscode folder, I highly recommend you to remove it from that folder, and put it in root directory of your project, and add .vscode folder on .gitignore, because you don't need to commit that folder on your repo, it's totally unnecessary,  [Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png)
- [ ] Try to make your pull request description more descriptive and give it a descriptive title
- [ ] Implement all changes I suggested in inline comments in your codes,

# [Optional changes]
- [ ] I would recommend you to use main, and footer tags for your respective web content
- [ ] Free signup button and subscribe button should use the same red colors from the original page
- [ ] The image in second article of top stories, should be well aligned (on small screens :point_right:  the alignment issue starts from screens below to 650px)
- [ ] In debate section, the black `VS` circle, should be black and circle, no lines travesal should be visible inside that button
- [ ] Starting from the debate section, you should add some margins on each side, currently the content of the page are touching the edge of screen in small sceens
- [ ] I advice you to try to use different images, not one image repetitively, :point_right:  this will make your clone looks more professional
- [ ] Your commit messages should be in imperative mode or present tense, you should have used `Fix the header section and adjust the border section of the subscribe and newsletter` instead of `fixed the header section and adjusted te border section of the subscribe and newsletter`, refer [to this article for more details on how to write a good && professional commit message](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

If anything seems to be unclear to you, don't hesitate to contact me for more clarification slack: `@Mugirase Emmanuel` | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Review by PhillipUg — APPROVED** [review]
# Status: Project Approved! 🥇 
Congratulations on the hard work and also for trying your best to implement all the changes requested by the previous reviewers. :+1:

#### NOTE: The following suggested changes are not mandatory but are nice to have and once implemented, will make your project look professional and nice 😃 

- [OPTIONAL]: The Navbar is still not yet quite there. Eliminate the space at the end towards the right.

| Original page      | Your page     |
| :------------- | :----------: |
|screenshot-www newsweek com-2020 10 26-15_23_42 | screenshot-raw githack com-2020 10 26-15_24_20 |

- [OPTIONAL]: Also, the section above `In the magazine` on medium screen size shouldn't be full screen, it should be centered in the middle.

| Original page | Your page |
| :------------- | :----------: |
|screenshot-www newsweek com-2020 10 26-15_32_16 | screenshot-raw githack com-2020 10 26-15_31_08 |

You can merge this branch into the master branch when you're done making the necessary changes and go on to your next project.

:bulb:  Do not forget to submit your project completion form after merging this branch.

All the best team,

For any assistance, you can contact me on slack as **Phillip Musiime**

**Pull request merged**